galette/templates/default/advanced_search.tpl

```diff
@@ -165,28 +165,68 @@
                 <ul id="fs_sortable" class="fields_list connectedSortable">
 {foreach from=$filters->free_search item=fs}
                     <li>
-                        <select name="free_logical_operator[]">
+                        <select id="logical_operator_selector" name="free_logical_operator[]">
```

Comment:
Since we're in a foreach; you may not use `@id` here because it must be unique on the page.